.github/workflows/ci.yml

```diff
@@ -20,8 +20,8 @@ permissions:
   contents: read
 
 env:
-  RUST_STABLE: 1.51
-  RUST_NIGHTLY: nightly-2021-05-10
+  RUST_STABLE: 1.55
+  RUST_NIGHTLY: nightly-2021-09-09
   CARGO_NDK: 2.4.1
```

Comment:
```
  # cmd: `shasum -a 256 cross`
  CROSS_CHECKSUM: 18a3599273773a7d1e333b413751f0adc1a83ec8045d060fee4f0639bd6baaf0
  # On targets like `*-linux-android*` and `wasm32-unknown-unknown`, reading the model
  # takes a considerable amount of time if it is compiled with the default opt-level
  # for tests. Therefore, we compile them with a higher opt-level to run them faster.
  OPT_TESTS_RUSTFLAGS: '-C opt-level=3 -C debug-assertions=yes -D warnings'
```

btw this can be removed beacuse we no longer run cross in the pr ci

also this can be removed in the bors + pr ci. I think I forgot to delete that when I was working on the cross integration.

```
CARGO_NDK: 2.4.1
ANDROID_PLATFORM_VERSION: 21
```

Reply:
#299